src/main/java/frc/robot/commands/RotateStorage.java

```diff
@@ -0,0 +1,24 @@
+package frc.robot.commands;
+
+import com.spikes2212.command.genericsubsystem.commands.smartmotorcontrollergenericsubsystem.MoveSmartMotorControllerGenericSubsystem;
+import com.spikes2212.control.FeedForwardSettings;
+import com.spikes2212.control.PIDSettings;
+import com.spikes2212.util.UnifiedControlMode;
+import frc.robot.subsystems.CoralJoint;
+
+public class RotateStorage extends MoveSmartMotorControllerGenericSubsystem {
+
+    private final CoralJoint coralJoint;
+
+    public RotateStorage(CoralJoint coralJoint, PIDSettings pidSettings, FeedForwardSettings feedForwardSettings,
```

Comment:
add a constructor which receives a Supplier<Double> as a setpoint in addition to this one